modules/popup/react/lib/usePopupStack.ts

```diff
@@ -40,23 +41,22 @@ import {PopupStack} from '@workday/canvas-kit-popup-stack';
  * useCloseOnEscape(stackRef, closePopup)
  */
 export const usePopupStack = <E extends HTMLElement>(
-  forwardRef?: React.RefObject<E>,
+  ref?: React.Ref<E>,
```

Comment:
I'm not 100% on what to do here. I really hate that `div` elements are `HTMLDivElement` instead of `HTMLElement` which causes all kinds of type incompatibilities when you use any tag other than a `div`.

The `PopupStack` is in charge of creating the element and it makes a `div`. That ref is never meant to be applied to a React element and this will throw an error if you even attempt it. The `RefObject` returned by this function will be used by all behavior hooks that are meant to be used with stacks and is the `RefObject` used by a React Portal.